ontologies/esrfet/ESRFET.owl

```diff
@@ -820,7 +759,16 @@
     <EquivalentClasses>
         <Class IRI="#LDDS-XAS"/>
         <ObjectIntersectionOf>
-            <Class IRI="#experimental_technique"/>
+            <Class IRI="http://purl.org/pan-science/PaNET/PaNET01196"/>
```

Comment:
Perhaps the connection to PANET can be moved to another PR because there are too many conflicts? So this PR could add the nexus submodule and ./devnotes/... What do you think?

Reply:
Also for PANET we could have a git submodule. But again there is the question of versioning.